libraries/hotstuff/qc_chain.cpp

```diff
@@ -525,14 +525,14 @@ namespace eosio { namespace hotstuff {
                fc_tlog(_logger, " === ${id} phase increment on proposal ${proposal_id}", ("proposal_id", vote.proposal_id)("id", _id));
                hs_proposal_message proposal_candidate;
 
-               if (_pending_proposal_block == NULL_BLOCK_ID)
+               if (_pending_proposal_block.empty())
                   proposal_candidate = new_proposal_candidate( p->block_id, p->phase_counter + 1 );
                else
                   proposal_candidate = new_proposal_candidate( _pending_proposal_block, 0 );
 
                reset_qc(proposal_candidate.proposal_id);
                fc_tlog(_logger, " === ${id} setting _pending_proposal_block to null (process_vote)", ("id", _id));
-               _pending_proposal_block = NULL_BLOCK_ID;
+               _pending_proposal_block = block_id_type{};
```

Comment:
This works and I slightly prefer.
```suggestion
               _pending_proposal_block = {};
```

Reply:
I actually went back and forth. I had it as `{}` first. I'll make the change.